src/parser/core/modules/Statuses.js

```diff
@@ -47,7 +47,7 @@ export default class Statuses extends Module {
 			if (ac.statusesApplied) {
 				ac.statusesApplied.forEach(st => {
 					if (st) {
-						this._statusToActionMap[st.id] = ac.id
+						this._statusToActionMap[st.id] = getDataBy(ACTIONS, 'id', ac.id)
```

Comment:
```suggestion
						this._statusToActionMap[st.id] = ac
```

Already got the action, don't need to look it up.